### ISSUE TITLE
Remove modules section from wiki

### ISSUE THREAD

**Issue by rodrigc** [jmmv/kyua#127]
The modules section in the Kyua wiki provides too low-level implementation details.
It makes the document too wordy.  Also, it is not accurate after latest refactorings of kyua code.
I suggest to remove it:

https://github.com/rodrigc/kyuawiki1/commit/a65069a5fc0ce0dd10013d249efa74d75fad59d2

**Comment by ngie-eign** [comment]
On Oct 10, 2014, at 12:28, Craig Rodrigues notifications@github.com wrote:

> The modules section in the Kyua wiki provides too low-level implementation details.
> It makes the document too wordy. Also, it is not accurate after latest refactorings of kyua code.
> I suggest to remove it:
> 
> rodrigc/kyuawiki1@a65069a

Hi Craig,

I agree that this makes more sense as developer documentation than end-user documentation.

Thanks!=

**Comment by jmmv** [comment]
Removed!
